src/sentry/models/integrations/__init__.py

```diff
@@ -1,20 +1,36 @@
 __all__ = (
     "DocIntegration",
+    "ExternalActor",
+    "ExternalIssue",
     "Integration",
     "IntegrationExternalProject",
+    "IntegrationFeature",
     "OrganizationIntegration",
     "PagerDutyService",
     "ProjectIntegration",
     "RepositoryProjectPathConfig",
+    "SentryApp",
+    "SentryAppComponent",
+    "SentryAppInstallation",
+    "SentryAppInstallationForProvider",
+    "SentryAppInstallationToken",
 )
 
 # REQUIRED for migrations to run.
 from sentry.types.integrations import ExternalProviders  # NOQA
 
 from .doc_integration import DocIntegration
+from .external_actor import ExternalActor
+from .external_issue import ExternalIssue
 from .integration import Integration
 from .integration_external_project import IntegrationExternalProject
+from .integration_feature import IntegrationFeature
 from .organization_integration import OrganizationIntegration
 from .pagerduty_service import PagerDutyService
 from .project_integration import ProjectIntegration
 from .repository_project_path_config import RepositoryProjectPathConfig
+from .sentry_app import SentryApp
```

Comment:
I'd be ok with making a separate `/src/sentry/models/sentry_apps/` directory but I think it make sense to leave them with the other Integration models.